CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,159 @@
+
+
+## Scope in which the pyOpenSci code of conduct is applied
+
+pyOpenSci community members are held to the standards outlined in this 
+code of conduct when interacting in the pyOpenSci Slack or GitHub 
+repositories, when interacting in-person at events where they could 
+represent pyOpenSci (this is most professional events), in physical spaces 
+with other pyOpenSci team members or collaborators, or in any Community 
+Initiatives (our fiscal sponsor) space.
+
+In addition, the pyOpenSci community and experience often extends outside 
+those spaces—pyOpenSci community members may go on walks together to get 
+lunch, attend meetups or conferences as a group, communicate on social 
+media, or interact with each other in other communities. 
+
+Abusive or unwelcoming behavior between community members still has a 
+profound impact on individuals and on the community when it happens beyond 
+our walls. The pyOpenSci Executive Director and the Advisory Council will 
+use our discretion when deciding whether to enforce this code of conduct 
+and potentially remove someone from the pyOpenSci community after reports 
+of such behavior happening outside of pyOpenSci, taking into account the 
+impact on the individual community members involved as well as the impact 
+on the community at large.
+
+```{note}
+When in doubt, please report unacceptable behavior to us. If someone's behavior make you feel unsafe within the pyOpenSci space, we want to support and help you. 
+```
+
+## Enforcement
+
+We’ve categorized unacceptable behavior into abuse and unwelcoming behavior in the section above.
+
+### Enforcement related to abusive behavior 
+
+We do not tolerate abusive behavior at pyOpenSci. 
+If we witness or receive a report of abusive behavior, we will 
+contact the perpetrator to have a conversation with them and verify 
+what has transpired. 
+
+We will follow this response protocol:
+
+If we verify abusive behavior:
+
+* They will be removed from the pyOpenSci community
+* If applicable, their employment with pyOpenSci will be terminated. 
+* Their Slack account will be deactivated, and 
+* Permissions will be removed from any pyOpenSci-related repositories and/or accounts. 
+
+The person will not be welcome in any physical or digital spaces covered by the pyOpenSci Code of Conduct.
+
+### Enforcement related to unwelcome behavior 
+If we verify unwelcome, but non-abusive behavior, we will have a 
+conversation with the person about their behavior. The goal of the conversation will be to ensure that they understand the expectation that they not repeat the behavior or other behaviors that would violate the Code of Conduct a second time. 
+
+### Response protocol steps
+The pyOpenSci Executive Director and the Executive Council will use the steps below to respond to reports of code of conduct violations.
+
+1. The report will be read carefully to determine whether there has been a code of conduct violation
```

Comment:
we might need to do steps 2 and 3 before determining if there has been a CoC violation

Reply:
good point. i've adjust this now.